SOTS_2019_WDR.tex

```diff
@@ -0,0 +1,502 @@
+    
+\end{checklist}
+\begin{center}
+\begin{tabu} to \textwidth { |[1pt]X[c]|[1pt]X[c]|[1pt]X[c]|[1pt] }
+\tabucline[1pt]{}
+Launch Control & Launch Tower & Minimum Safe Distance \\
+\tabucline[1pt]{}
+\vspace{1in} & & \\
+\tabucline[1pt]{}
+\end{tabu}
+\end{center}
+\setcounter{checklistnum}{0}
+% end prior to start section -- }}}
+
+% begin Fill System Check section -- {{{
+\subsection{Nominal Procedure}
+\begin{checklist}
+    \item \primary{}: Confirm that the following valves are initially closed:
+        \begin{checklist}
+        \item Cylinder Valve
+        \item Remote Fill Valve
+        \item Parallel Fill Valve
+        \item Series Fill Valve
+        \item Line Vent Valve
+        \item Parallel Vent Valve
+    \end{checklist}
+    \item \primary{}: Confirm that the ignition connectors are disconnected from the rocket.
+    \item \control{}: Power on the client-side RLCS box.
+    \item \control{} and \secondary: Confirm that the following actuators fail to move:
+    \begin{checklist}
+        \item Remote Fill Valve
+        \item Line Vent Valve
+        \item Remote Disconnect
+        \item Injector Valve
+    \end{checklist}
+    \item \secondary: Confirm that the voltage across the ignition connectors is 0 V.
+    \item \ops: Give the system control key to \control{}.
+    \item \control: Confirm that all actuator controls are in the off state:
+    \begin{checklist}
+        \item Remote Fill Valve
+        \item Line Vent Valve
+        \item Remote Disconnect
+        \item Tank Vent Valve
+        \item Primary Ignition
+        \item Secondary Ignition
+        \item Injector Valve
+    \end{checklist}
+    \item \control{}: Engage the key switch and enable actuators.
+    \item \control{} and \secondary: Confirm that all actuators actuate as intended:
+    \begin{checklist}
+        \item Remote Fill Valve
+        \item Line Vent Valve
+        \item Remote Disconnect
+        \item Tank Vent Valve
+        \item Injector Valve
+    \end{checklist}
+    \item \control{} and \secondary: Confirm that the ignition voltage is 12 V when the ignition button is fired:
+    \begin{checklist}
+        \item Primary Ignition
+        \item Secondary Ignition
+    \end{checklist}
+    \item \control: Confirm that all DAQ readings are displaying appropriately.
+    \item \ops{}: Record the resting DAQ values:
+
+    \begin{tabu} to \textwidth { |[1pt]X[c]|[1pt]X[c]|[1pt]X[c]|[1pt]X[c]|[1pt] }
+    \tabucline[1pt]{}
+    [M] Dry Mass (lbs) & [P1] Supply Pressure (psi) & [P2] Fill Line Pressure (psi) & [P3] Oxidizer Tank Pressure (psi) \\
+    \tabucline[1pt]{}
+    \vspace{1in} & & & \\
+    \tabucline[1pt]{}
+    \end{tabu}
+    
+    \item \control: Remove the system control key and give it to \ops.
+    \item \primary: Arm the payload using the transponder.
+    \item \primary: Arm recovery avionics using the magnetic switches
```

Comment:
This will be part of _launch_, but won't be part of WDR. We don't assemble recovery for this test.